### ISSUE TITLE
Issue with Subsections in Grid Format Displayed as 'Orphaned Activities' in Moodle 4.5+

### ISSUE THREAD

**Issue by studio-11** [gjbarnard/moodle-format_grid#230]
**Describe the bug**
When using the Grid format in a Moodle course, attempting to create a subsection within an existing section results in the subsection title being displayed as "orphaned activities." Additionally, the following message appears:
"This section and its content are not part of the course structure and are not visible to students. To use any of this content, move it to a different section."
Even after trying to move content to a different section, the issue persists. The total number of sections that can be created is well above the number of sections currently in the course.
**To Reproduce**
Steps to reproduce the behaviour:

Go to a Moodle course configured with the Grid format.
Create a new section.
Attempt to create a subsection within this new section.
Observe the title and message of the subsection.

**Expected behaviour**
The subsection should display a user-defined title and behave like a standard part of the course, appearing properly within the course structure and being visible to students (unless hidden explicitly).
**Screenshots**

**Versions (please complete the following information):**
 Moodle: 4.5+
Format: Grid version 405.0.1
Browser: Chrome Version 131.0.6778.86

**Additional context**
The total number of possible sections in the course exceeds the number of sections currently created. This issue seems specific to the Grid format and does not occur in other course formats. Let me know if further logs or settings need to be shared.
![Screenshot 2024-12-03 at 17 48 56](https://github.com/user-attachments/assets/17b48bcc-56f2-4818-9179-115f61a20587)

**Comment by gjb2048** [comment]
@studio-11 Does the add subsection option look like this https://www.youtube.com/watch?v=Tqk3gjuAYHw ?

**Comment by studio-11** [comment]
yes it does. here is my video screenrecord : https://youtu.be/x4mlCDnyVgk

**Comment by gjb2048** [comment]
@studio-11 Thank you.

**Comment by gjb2048** [comment]
@studio-11 Fixed in development repository.  Fix will be in the next release.

**Comment by davidscotson** [comment]
Is this bug fix merged? I can't see it in the main branch.

**Comment by gjb2048** [comment]
@davidscotson As stated in https://github.com/gjbarnard/moodle-format_grid/issues/230#issuecomment-2518113789 - the fix will be in the next release.  The development repository is private to prevent inadvertent installation of development code on production servers and the subsequent 'this isn't working, panic, panic, panic' issues.

**Comment by FranScicluna** [comment]
When will the next release be published please?

**Comment by gjb2048** [comment]
@FranScicluna Probably within the next month or two.  Still learning about the sub-section changes with Collapsed Topics and that is also feeding into this.

**Comment by FranScicluna** [comment]
Okay, thank you :)
